data_steward/resource_files/fields/_unit_mapping.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
Good idea to prefix this with '_': it will be recognized as internal by `resources.cdm_schemas()`. :)